Benqi-Agents/DelegationMonitor/src/agent.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  Finding,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+  ethers,
+  HandleTransaction,
+} from "forta-agent";
+import { BigNumber, providers } from "ethers";
+
+import util from "./utils";
+
+const Qi_CONTRACT: string = "0x8729438EB15e2C8B576fCc6AeCdA6A148776C0F5";
+const AMOUNT_THRESHOLD: BigNumber = BigNumber.from(10 ** 6); // 1M USD
+
+export const createFinding = (
+  delegator: string,
+  fromDelegate: string,
+  toDelegate: string,
+  balance: BigNumber
+): Finding => {
+  return Finding.fromObject({
+    name: "Delegations Monitor",
+    description: "Detect user with a huge balance delegating their votes",
+    alertId: "BENQI-2",
```

Comment:
I think `name` and `description` can be improved here, the name isn't very understandable.  I would suggest something like: 
```
    name: "Large votes delegation detected",
    description: "`DelegateChanged` event was emitted and the delegator holds a large balance",
    alertId: "BENQI-2",
```
feel free to improve this to what seems good to you